generators/app/templates/gulp/tasks/vendor.js

```diff
@@ -2,11 +2,30 @@
 
 var fs = require('fs');
 var path = require('path');
+var chalk = require('chalk');
 
 var bundleVendorTask = function (gulp, plugins, config) {
   var fileListPath = path.join(config.src.base, 'scripts/vendor.json');
   var getFileList = function () {
-    return JSON.parse( fs.readFileSync(fileListPath) );
+    var fileList = '';
```

Comment:
readFileSync doesn't return string by default, either make it return string or don't define fileList as string